pylint/checkers/variables.py

```diff
@@ -336,6 +337,64 @@ def _assigned_locally(name_node):
 
     }
 
+
+NamesConsumerAtomic = collections.namedtuple("NamesConsumerAtomic",
```

Comment:
Maybe we can rename it to ScopeConsumer since it is used mostly to hold names belonging to a scope?